src/pages/registration/resource_type_tab/components/SearchContainerField.tsx

```diff
@@ -200,8 +197,8 @@ interface ContainerAndLevelTextProps {
 const ContainerAndLevelText = ({ registration }: ContainerAndLevelTextProps) => {
   const { t } = useTranslation();
 
-  const publicationContext = registration.entityDescription?.reference?.publicationContext;
-  const publisherId = publicationContext?.publisher?.id ?? '';
+  const publicationContext = registration.publishingDetails;
+  const publisherId = publicationContext?.id ?? '';
```

Comment:
Tidligere menere jeg publisher var i en egen property `publisher` mens tidsskrift bare var på rota som `id`. Er både publisher og journal i `id` nå?

Reply:
Ser det mangler noe data lokalt vs i dev her iaf. Men mistenkelig lite på det vi har i dev også, så litt usikker på hva ståe er her 🤔 
![bilde](https://github.com/user-attachments/assets/670f217e-bbf7-4bcc-8e98-48049b3b58b2)
![bilde](https://github.com/user-attachments/assets/d8b3b129-132d-4d57-96c6-7190a3ad52f0)

Reply:
Ser ut som ternary-sjekken på om options skal vise `year-and-contributors` eller `publisher-and-level` ruter oss hit. Dermed kommer vi bare hit når vi selv mener vi skal ha publisher i details, så da burde det stemme fra frontend som jeg forstår det. Men publisher er utelatt fra søketreffet i det første tilfelle der da, så det er et konkret eksempel til Sondre om jeg ikke misser noe